<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration_win</class>
 <widget class="QDialog" name="Registration_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton 
{     
background-color: #67BA80;  /* Цвет фона кнопки */    
color: white;  /* Цвет текста */    
border-radius: 5px;  /* Скругленные углы */ 
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>202</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_registr"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_login_registr"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_password_regist"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_repit_pass_auto"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_registration">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="text">
       <string>Авторизироваться</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
